cou/utils/openstack.py

```diff
@@ -149,6 +150,8 @@
     "lunar": "antelope",
 }
 
+SOME_TABLE = {"focal-yoga": True}
```

Comment:
Did you miss to remove this line?

Reply:
Indeed :laughing: Thanks for catching it.